docs/products/postgresql/howto/pg-object-size.rst

```diff
@@ -0,0 +1,98 @@
+    (6 rows)
+    
+    testdb2=> \l+ testdb2
+                                                    List of databases
+      Name   |  Owner   | Encoding |   Collate   |    Ctype    | Access privileges | Size  | Tablespace | Description 
+    ---------+----------+----------+-------------+-------------+-------------------+-------+------------+-------------
+     testdb2 | avnadmin | UTF8     | en_US.UTF-8 | en_US.UTF-8 |                   | 66 MB | pg_default | 
+    (1 row)
+
+An alternative to get the database size is to use the ``pg_database_size`` function::
+
+    testdb2=> select pg_database_size('testdb2'); 
+     pg_database_size 
+    ------------------
+             68895523
+    (1 row)
+    
+    testdb2=> select pg_size_pretty(pg_database_size('testdb2')); 
+     pg_size_pretty 
+    ----------------
+     66 MB
+    (1 row)
+
+As shown on the examples above the output for the database **testdb2** size is the same, though the ``pg_database_size`` function will return the database size in bytes, so we used the ``pg_size_pretty`` function to return an easy to read output, we will keep using it in the coming examples. 
+
+Get table size
+--------------
+In order to get the table size the ``\dt+ [ pattern ]`` command can be used::
+
+    testdb2=> \dt+ mytable1
+                                           List of relations
+       Schema    |   Name   | Type  |  Owner   | Persistence | Access method | Size  | Description 
+    -------------+----------+-------+----------+-------------+---------------+-------+-------------
+     test_schema | mytable1 | table | myowner  | permanent   | heap          | 14 MB | 
+    (1 row)
+
+
+The function ``pg_table_size`` can be used to get the table size::
+
+    testdb2=> select pg_size_pretty(pg_table_size('mytable1')); 
+     pg_size_pretty 
+    ----------------
+     14 MB
+    (1 row)
+
+Get index size
+--------------
+Individual index size can be retrieved using the command ``\di+ [ pattern ]``::
+
+    testdb2=> \di+ mytable1_indx1 
+                                                    List of relations
+       Schema    |      Name      | Type  |  Owner   |  Table   | Persistence | Access method |  Size  | Description 
+    -------------+----------------+-------+----------+----------+-------------+---------------+--------+-------------
+     test_schema | mytable1_indx1 | index | myowner  | mytable1 | permanent   | btree         | 912 kB | 
+    (1 row)
+
+The function ``pg_indexes_size`` can be used to query the size for **all the indexes** that belong to a table::
+
+    testdb2=> select pg_size_pretty(pg_indexes_size('mytable1')); 
+     pg_size_pretty 
+    ----------------
+     912 kB  
+    (1 row)  
+    
+Get table and its indexes size
+--------------------------------
+It is possible to get the disk usage for a table and its corresponding indexes, the function ``pg_total_relation_size`` computes the total disk space used by the specified table, including all indexes and TOAST data::
```

Comment:
To get disk space usage for a table and its indexes, you can use the ``pg_total_relation_size`` function, which computes the total disk space used by the table, all its indices, and TOAST data.